src/components/Checkout/DonateRepaymentsToggle.vue

```diff
@@ -1,29 +1,69 @@
 <template>
-	<div v-if="showToggle" class="donate-repayments-toggle">
-		<kv-checkbox
-			class="donate-repayments-label"
-			id="donate-repayments"
-			:checkbox-right="true"
-			v-if="!myDonateRepayments"
-			v-model="donateRepayments"
-			@change="toggleDonateRepayments"
+	<div>
+		<div
+			v-if="showToggle"
+			class="tw-mt-2"
+			:class="{ 'tw-flex' : checkboxRight }"
 		>
-			<span
+			<label
+				v-if="checkboxRight"
+				for="donateRepayments"
 				id="donate-repayments-tooltip"
-				class="tw-text-small tw-text-link tw-font-medium"
+				class="tw-flex-1 tw-pt-0.5 md:tw-pt-0 tw-text-base
+					tw-font-medium tw-text-left md:tw-text-right tw-mr-2
+					tw-cursor-pointer"
+				@click="toggleCheckbox"
 			>
 				Donate loan repayments instead?
-			</span>
-		</kv-checkbox>
-		<kv-tooltip controller="donate-repayments-tooltip" theme="mint">
-			<template #title>
-				Thanks for your support!
-			</template>
-			When you check this box, repayments go back to Kiva in the form of donations,
-			helping us cover operating costs and reach even more borrowers worldwide.
-			<br>
-			Repayments from selected loans will not be added back to your account as Kiva credit.
-		</kv-tooltip>
+			</label>
+			<kv-checkbox
+				class="tw-relative tw-cursor-pointer md:tw-text-right kv-checkbox"
+				:class="{ 'tw-pt-0 ' : checkboxRight }"
+				id="donateRepayments"
+				ref="donateRepayments"
+				v-if="!myDonateRepayments"
+				v-model="donateRepayments"
+				@change="toggleDonateRepayments"
+			>
+				<span
+					id="donate-repayments-tooltip"
+					class="tw-text-small tw-font-medium"
+					:class="{ 'tw-sr-only' : checkboxRight }"
+				>
+					Donate loan repayments instead?
+				</span>
+			</kv-checkbox>
+			<kv-lightbox
+				:visible="isLightboxVisible"
+				title="Donate your loan repayments"
+				@lightbox-closed="closeLightbox"
+			>
+				When you check this box, repayments go back to Kiva in the form of donations,
+				helping us cover operating costs and reach even more borrowers worldwide.
+				Repayments from selected loans will not be added back to your account as Kiva credit.
+			</kv-lightbox>
+
+			<kv-tooltip v-if="!checkboxRight" controller="donate-repayments-tooltip" theme="mint">
```

Comment:
Since `checkboxRight` is always true, I don't think this will ever be used?

Reply:
I left this in there so we had the ability to flip the checkbox to it's previous state by just flipping a boolean. But perhaps cleaning it up is a better option?

Reply:
Will be removed in my next commit.